bundles/org.openhab.binding.openweathermap/README.md

```diff
@@ -77,33 +98,49 @@ Once the parameter `forecastDays` will be changed, the available channel groups
 | current          | time-stamp           | DateTime             | Time of data observation.                                               |
+| current          | sunrise              | DateTime             | Sunrise time of current day. Only available in the One Call API         |
+| current          | sunset               | DateTime             | Sunset  time of current day. Only available in the One Call API         |
 | current          | condition            | String               | Current weather condition.                                              |
 | current          | condition-id         | String               | Id of the current weather condition. **Advanced**                       |
 | current          | icon                 | Image                | Icon representing the current weather condition.                        |
 | current          | icon-id              | String               | Id of the icon representing the current weather condition. **Advanced** |
 | current          | temperature          | Number:Temperature   | Current temperature.                                                    |
 | current          | apparent-temperature | Number:Temperature   | Current apparent temperature.                                           |
 | current          | pressure             | Number:Pressure      | Current barometric pressure.                                            |
-| current          | humidity             | Number:Dimensionless | Current atmospheric humidity.                                           |
+| current          | humidity             | Number:Dimensionless | Current atmospheric humidity.  
+| current          | dew-point            | Number:Temperature   | Current dew-point. Only available in the One Call API                   |
 | current          | wind-speed           | Number:Speed         | Current wind speed.                                                     |
 | current          | wind-direction       | Number:Angle         | Current wind direction.                                                 |
 | current          | gust-speed           | Number:Speed         | Current gust speed. **Advanced**                                        |
 | current          | cloudiness           | Number:Dimensionless | Current cloudiness.                                                     |
 | current          | rain                 | Number:Length        | Rain volume of the last hour.                                           |
 | current          | snow                 | Number:Length        | Snow volume of the last hour.                                           |
 | current          | visibility           | Number:Length        | Current visibility.                                                     |
+| current  | uvindex    | Number    | Current UV Index. Only available in the One Call API      |
```

Comment:
Can you align this table row

Reply:
Done (also for some other misaligned table rows)